torch_xla/csrc/helpers.cpp

```diff
@@ -205,6 +205,18 @@ xla::XlaOp XlaHelpers::ReshapeToRank(const xla::XlaOp& input,
   return xla::Reshape(input, dimensions);
 }
 
+std::vector<xla::int64> XlaHelpers::MakeTransposePermutation(xla::int64 dim0,
+                                                             xla::int64 dim1,
+                                                             xla::int64 rank) {
+  xla::int64 canonical_dim0 =
+      XlaHelpers::GetCanonicalDimensionIndex(dim0, rank);
```

Comment:
No need for XlaHelpers:: ... we are in XlaHelpers now.

Reply:
Done.